.github/workflows/build.yml

```diff
@@ -252,7 +252,14 @@ jobs:
         # On Mac/ARM, the JSON output disagrees in the 16th and 17th digit
         #  with the JSON output on Linux/Intel and Mac/Intel.
         if [ ${{ matrix.arch }} = "mac-arm64" ] ; then
-          find tests -name '*.json' | xargs rm
+           find tests -name '*.json' | xargs rm
+        fi
+
+        # Another ugly hack: remove the *.errout file generated for the CBDSP
```

Comment:
It might be better if there is at least one architecture that we do NOT remove files for.
Right now this would be Linux.
So it would be slightly better if we remove the Mac and Windows ones, IMHO.
No strong feelings though.

Reply:
Hmm, good point, I didn't think about consistency. I disabled the Linux check because in this case, it was Linux that behaved as an outlier. No strong feelings on this either, though.

Reply:
Hmm... in that case, case we remove the Mac and Windows files instead?

Reply:
Done.